src/app/components/get-this-title/get-this-title.html

```diff
@@ -16,10 +16,14 @@
             </div>
             <div class="print-submenu">
                 <h3>Place an Order</h3>
-                <p>
+                <p if="model.bookstore_link">
                     <a href="{model.bookstore.link}" class="btn">Bookstore Orders</a>
                     Order through NACSCORP
                 </p>
+                <p if="!model.bookstore_link">
+                    <a href="/bulk-order?{model.slug}" class="btn">Bulk Orders</a>
+                    Fill out our form to order in bulk for your school.
+                </p>
```

Comment:
Looks like `<else>` requires the standalone `<if>` tag. I'm rewriting some `if`s.